white_space.py

```diff
@@ -0,0 +1,41 @@
+"""This module does that automatially checks for pep8 format."""
+# !/usr/bin/env python
+
+import os
+import autopep8
+import sys
+
+
+BASE_DIR = os.path.abspath(__file__)
+
+
+def format_file(path):
+    """This is the main function that does the work."""
```

Comment:
This docstring does not explain what the actual work that this function does. The docstring should explain the function's task, so that reading the source is not confusing. Check this - https://www.python.org/dev/peps/pep-0257/